guides/users/user-metadata.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: User Metadata
+---
+
+Hanko allows for defining arbitrary user metadata. Metadata can be categorized into
+three types that differ as to how they can be accessed and modified:
+
+| Metadata type | Public API                   | Admin API             |
+|---------------|------------------------------|-----------------------|
+| Private       | No read or write access      | Read and write access |
+| Public        | Read access                  | Read and write access |
+| Unsafe        | Read access and write access | Read and write access |
+
+Each metadata type is currently limited to 3000 characters.
```

Comment:
I'm not sure if we want to document, that the JSON structure characters (e.g. `{`, `}`) are included in the limit. What do you think?